bin/cli.js

```diff
@@ -11,9 +11,14 @@ const cli = new Cli()
 program
   .version(pkgJson.version)
   .arguments('<cmd>')
-  .action((cmd) => {
+  .option(
+    '-b, --blackduck <directory>',
```

Comment:
As I mentioned before, I don't think we want to use the term `blackduck` in the open-source version. The format isn't actually specific to blackduck. Something more generic, like `dependencies` is probably more appropriate.